### DISCUSSION TITLE
Addon manager

### DISCUSSION THREAD

**Discussion by Emile874** [community/community#56198] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I update FreeCAD 20.1 to FreeCAD 20.2 and my A2Plus workbench is gone. Went to the addon manager and it's not there !! Can someone help, please?

**Comment by HappyMicky0317** [comment, marked as answer]
It's possible that the A2Plus workbench might not be compatible with FreeCAD 20.2. You could try reinstalling the A2Plus workbench by downloading it from the FreeCAD website and placing it in the appropriate directory. Alternatively, you could try reaching out to the FreeCAD community on their forum or mailing list for further assistance.

**Reply by HappyMicky0317** [reply]
I also faced this issue😢

**Comment by Emile874** [comment]
Hi, thanks for the help !! I re-installed freeCAD 20.1 and A2Plus workbench from the freeCAD website. Did the update to 20.2 and no problem.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Emile874! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
